dist/valid-head.js

```diff
@@ -0,0 +1,8 @@
+var valid_elements = ['TITLE', 'META', 'LINK', 'SCRIPT', 'STYLE', 'BASE', 'NOSCRIPT', 'TEMPLATE']
+var elems = document.head.getElementsByTagName("*");
+for (var i=0; i<elems.length;i++) {
+    if (!valid_elements.includes(elems[i].tagName)) {
+        return true;
```

Comment:
It might be useful metadata to return the list of invalid elements for further analysis/exploration.